Makefile

```diff
@@ -131,6 +138,11 @@ copy_applet:
 	cp ${APPLET_PATH}/trusted_applet.elf ${CURDIR}/trusted_os/assets/
 	cp ${APPLET_PATH}/trusted_applet.sig ${CURDIR}/trusted_os/assets/
 
+create_dummy_applet:
+	mkdir -p $(DIR)/assets
+	touch $(DIR)/assets/trusted_applet.elf
```

Comment:
I think `touch` will just update the `mtime` if the file already exists.
You'll probably have to either `-rm` these files first, or maybe do a `cp /dev/null ...` or `truncate` command instead?

Reply:
Good point, done.